doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1550 @@
+  * Fixed the missing log source when passing a direct log call to the nRF Cloud logging backend.
+    This caused the log parser to incorrectly use the first declared log source with direct logs when using dictionary mode.
+
+* :ref:`lib_fota_download` library:
+
+  * Fixed an issue where the download client instance did not use native TLS although the :kconfig:option:`CONFIG_FOTA_DOWNLOAD_NATIVE_TLS` Kconfig option was enabled.
+
+* :ref:`lib_mqtt_helper` library:
+
+  * Added the function :c:func:`mqtt_helper_msg_id_get` to get valid message IDs for MQTT messages.
+  * Updated the :kconfig:option:`CONFIG_MQTT_HELPER_PROVISION_CERTIFICATES` Kconfig option to depend on :kconfig:option:`CONFIG_TLS_CREDENTIALS` instead of specific boards.
+
+* :ref:`lib_nrf_provisioning` library:
+
+  * Added support for the ``SO_KEEPOPEN`` socket option to keep the socket open even during PDN disconnect and reconnect.
+  * Updated the check interval logging to use INF to improve customer experience.
+
+* :ref:`lib_nrf_cloud_alert` library:
+
+  * Updated to use INF log level when cloud side changes the alert enable flag.
+
+Libraries for NFC
+-----------------
+
+* Added an experimental serialization of NFC tag 2 and tag 4 APIs.
+* Fixed a potential issue with handling data pointers in the function ``ring_buf_get_data()`` in the :file:`platform_internal_thread` file.
+
+nRF RPC libraries
+-----------------
+
+* Added:
+
+  * An experimental serialization of Openthread APIs.
+  * The logging backend that sends logs through nRF RPC events.
+
+* Updated the internal Bluetooth serialization API and Bluetooth callback proxy API to become part of the public NRF RPC API.
+
+Other libraries
+---------------
+
+* Added the :ref:`nrf_compression` library with support for the LZMA decompression.
+
+* :ref:`lib_date_time` library:
+
+  * Added:
+
+    * A retry feature that reattempts failed date-time updates up to a certain number of consecutive times.
+    * The Kconfig options :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` to control whether and how many consecutive date-time update retries may be performed, and :kconfig:option:`CONFIG_DATE_TIME_RETRY_INTERVAL_SECONDS` to control how quickly date-time update retries occur.
+
+  * Fixed a bug that caused date-time updates to not be rescheduled under certain circumstances.
+
+* :ref:`lib_ram_pwrdn` library:
+
+  * Added support for the nRF54L15 SoC.
+
+* :ref:`dult_readme` library:
+
+  * Added DULT motion detector feature.
+    You can enable this feature using the :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` Kconfig option.
+
+Security libraries
+------------------
+
+* :ref:`nrf_security_readme` library:
+
+  * Added the :kconfig:option:`CONFIG_PSA_WANT_ALG_SP800_108_COUNTER_CMAC` Kconfig option to key derivation function configurations in :ref:`nrf_security_driver_config`.
+    The Kconfig option enables support for the derivation function SP 800-108r1 CMAC in counter mode, which is supported by the nrf_cracen driver.
```

Comment:
Link to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-18192/nrf/libraries/security/nrf_security/doc/drivers.html#cracen-driver instead of nrf_cracen?